packages/grid/x-data-grid/src/locales/frFR.ts

```diff
@@ -144,12 +144,12 @@ const frFRGrid: Partial<GridLocaleText> = {
   rowReorderingHeaderName: 'Positionnement des lignes',
 
   // Aggregation
-  // aggregationMenuItemHeader: 'Aggregation',
-  // aggregationFunctionLabelSum: 'sum',
-  // aggregationFunctionLabelAvg: 'avg',
-  // aggregationFunctionLabelMin: 'min',
-  // aggregationFunctionLabelMax: 'max',
-  // aggregationFunctionLabelSize: 'size',
+  aggregationMenuItemHeader: 'Aggrégation',
```

Comment:
```suggestion
  aggregationMenuItemHeader: 'Agrégation',
```

https://www.linternaute.fr/dictionnaire/fr/definition/agregation/